dom0/sd-proxy.sls

```diff
@@ -28,12 +28,4 @@ sd-proxy:
         - sd-{{ sdvars.distribution }}
     - require:
       - qvm: sd-whonix
-      - qvm: sd-small-{{ sdvars.distribution }}-template
-
-# Permit the SecureDrop Proxy to manage Client connections
-sd-proxy-dom0-securedrop.Proxy:
-  file.prepend:
-    - name: /etc/qubes-rpc/policy/securedrop.Proxy
-    - text: |
-        sd-app sd-proxy allow
-        @anyvm @anyvm deny
+      - qvm: sd-small-{{ sdvars.distribution }}-template
```

Comment:
did a search (https://github.com/freedomofpress/securedrop-workstation/search?q=qubes-rpc) and looks like there are potentially other places we can clean up old rpc policy code (outside of this pr or as part of this PR if you'd like me to test and review tomorrow)

Reply:
Which places are you thinking? I just took another quick spin as well and didn't spot any other obvious removal/refactor candidates (though the overall `sls` organization/naming is IMO _very_ much a candidate for refactoring).

Reply:
e.g. https://github.com/freedomofpress/securedrop-workstation/blob/c19461996b5af7ea9c32c9e1bd01a646709b5801/dom0/sd-clean-all.sls#L79-L80 but let me pay a closer look at what's going on here...

Reply:
I think the `securedrop.Log` line could be removed, it looks like this policy has fully been moved into the new policy format. The `securedrop.Proxy` policy still exists as a standalone file so I think we need to keep that one.

(As I understand it, Qubes 4.1 still executes files in the old format _first_, and some policy files were kept in the old format to ensure that our policies take precedence; @eaon can correct me here if I'm wrong.)

Reply:
I think the fact that we aren't sure means we should capture this in an issue and avoid making the change today as an add-on to this PR since everything else looks good. I'll follow-up by creating an issue and also capture the sls clean up idea.

Reply:
Here's the issue we can capture the rest of the conversation in so that it's less buried: https://github.com/freedomofpress/securedrop-workstation/issues/818